WordPress/Classes/ViewRelated/Blog/Site Picker/BlogList/SiteIconViewModel.swift

```diff
@@ -15,7 +15,7 @@ struct SiteIconViewModel {
 
         var width: CGFloat {
             switch self {
-            case .small: 24
+            case .small: 28
```

Comment:
I tested other places where it is used – nav bar in posts, etc – and it looks good with a new size.